src/vs/workbench/contrib/positronNotebook/browser/notebookCells/NotebookCellActionBar.tsx

```diff
@@ -44,8 +44,7 @@ export function NotebookCellActionBar({ cell, children, isHovered }: NotebookCel
 
 	// Filter actions based on cell conditions
 	const cells = instance.cells.get();
-	const cellIndex = cells.indexOf(cell);
-	const cellInfo = createCellInfo(cell, cellIndex, cells.length);
+	const cellInfo = createCellInfo(cell, cell.index, cells.length);
```

Comment:
I think we should either have this interface be exclusively just the `cell` and then the internals can pull out the helpers, or we just pass the raw info without reference to the cell object itself.